components/data-table/row-actions.jsx

```diff
@@ -57,13 +57,19 @@ const DataTableRowActions = createReactClass({
 		/**
 		 * `Dropdown` options. See `Dropdown`.
 		 */
-		options: PropTypes.array.isRequired,
+		options: PropTypes.array,
+		/**
+		 * A `Dropdown` component. The props from this drop will be merged and override any default props.
+		 * **Note:** onAction will not be overridden, both `DropDown`'s onSelect and onAction will be called with appropriate parameters
```

Comment:
We probably need a console error here.  See https://github.com/salesforce/design-system-react/blob/master/components/input/check-props.js#L61 for an example of `onlyOneOfProperties`.

Reply:
Do we need this?  onAction wasn't required before, and they behave differently.  onAction calls with the row item as a param whereas onSelect only provides the dropdown action.

Reply:
Great question! Let's replace the note part with sharing the difference between the two callbacks and document this topic. List out the parameters and such. I think that will also help down the road with maintainability, when someone (like myself) says, why do we need both of these--without looking at the source code.